pages/index.tsx

```diff
@@ -12,7 +97,24 @@ const Home: NextPage = () => {
       </Head>
 
       <main>
-        <p>Posts go here</p>
+        {posts.length > 0 ? (
+          posts.map((post: any) => (
```

Comment:
In nextJS, you should be using getStaticProps for server side rendering of HTML:

https://nextjs.org/docs/basic-features/data-fetching/get-static-props